pkg/controller/plan/adapter/ocp/client.go

```diff
@@ -143,23 +143,31 @@ func (r *Client) PreTransferActions(vmRef ref.Ref) (ready bool, err error) {
 
 	// Check if VM export exists
 	vmExport := &export.VirtualMachineExport{}
-	err = r.sourceClient.Get(context.Background(), client.ObjectKey{Namespace: vmRef.Namespace, Name: vmRef.Name}, vmExport)
+	err = r.sourceClient.Get(context.Background(), k8sclient.ObjectKey{Namespace: vmRef.Namespace, Name: vmRef.Name}, vmExport)
+
 	if err != nil {
 		if !k8serr.IsNotFound(err) {
 			r.Log.Error(err, "Failed to get VM-export.", "vm", vmRef.Name)
 			return true, liberr.Wrap(err)
 		}
+
+		var tokenTTLDuration *metav1.Duration
+		if settings.Settings.CDIExportTokenTTL > 0 {
+			tokenTTLDuration = &metav1.Duration{Duration: time.Duration(settings.Settings.CDIExportTokenTTL) * time.Minute}
+		}
+
 		// Create VM export
 		vmExport = &export.VirtualMachineExport{
-			TypeMeta: meta.TypeMeta{
+			TypeMeta: metav1.TypeMeta{
 				Kind:       "VirtualMachineExport",
 				APIVersion: "kubevirt.io/v1alpha3",
 			},
-			ObjectMeta: meta.ObjectMeta{
+			ObjectMeta: metav1.ObjectMeta{
 				Name:      vmRef.Name,
 				Namespace: vmRef.Namespace,
 			},
 			Spec: export.VirtualMachineExportSpec{
+				TTLDuration: tokenTTLDuration,
```

Comment:
is it omittedwhen `tokenTTLDuration` is nil?

Reply:
indeed

Reply:
ack